radicle-httpd/src/api/v1/delegates.rs

```diff
@@ -62,3 +62,29 @@ async fn delegates_projects_handler(
 
     Ok::<_, Error>(Json(projects))
 }
+
+#[cfg(test)]
+mod routes {
+    use crate::api::v1::utils::{get_response, json, HEAD};
+
+    #[tokio::test]
+    async fn test_delegates_projects() {
+        let body = get_response(
+            "/v1/delegates/did:key:z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi/projects",
```

Comment:
notice `delegates` route takes `did`, do we want to change that so we can take `<pk>` instead of `did:key:<pk>`? there are other places in api which takes plain `<pk>`, please advise where each should be used.

Reply:
We should always take DIDs here, in case we want to later introduce another key type.

Reply:
what about other routes? e.g.[ `node_handler`](https://github.com/radicle-dev/heartwood/blob/master/radicle-httpd/src/api/v1/node.rs#L20) takes `NodeId`, correct usage?

Reply:
That's correct yeah.